rskj-core/src/main/java/co/rsk/config/RemascConfigFactory.java

```diff
@@ -34,24 +42,57 @@ public class RemascConfigFactory {
     private static final Logger logger = LoggerFactory.getLogger("RemascConfigFactory");
 
     private ObjectMapper mapper;
-    private String configPath;
+    private final Map<String, RemascConfig> remascConfigMap = new ConcurrentHashMap<>();
 
     public RemascConfigFactory(String remascConfigFile) {
```

Comment:
the initial idea was to fail earlier, and it seems that in the current style of its usage `RemascConfigFactory` is instantiated for each call of `getContractForAddress()`. See `org.ethereum.vm.PrecompiledContracts#getContractForAddress`. 

Do you think we could create an instance of `RemascConfigFactory` in the `PrecompiledContracts`'s ctor?

Reply:
Addressed